cfme/containers/project.py

```diff
@@ -111,11 +111,9 @@ def step(self):
         self.prerequisite_view.entities.get_entity(name=self.obj.name,
                                                    surf_pages=not search_visible,
                                                    use_search=search_visible).click()
-        self.view.toolbar.view_selector.select("Summary View")
 
     def resetter(self):
-        if (self.appliance.version.is_in_series('5.9') and
```

Comment:
I don't think we need to check versions anymore. 5.9 and 5.10 both has "Summary" view.